client/js/components/GameView/Card.vue

```diff
@@ -35,11 +35,11 @@ export default {
   name: 'Card',
   props: {
     suit: {
-      type: Number,
+      type: [Number, String],
```

Comment:
This definitely has a smell to it, feels like it should be one type. If suit is no longer always required, this should not be a required prop. Instead something like this:
```
suit: {
      type: Number,
```
which will default to `undefined` I think?